src/components/FilterTags/FilterTags.js

```diff
@@ -154,23 +153,23 @@ const FilterTags = () => {
 
   return (
     <div className="p-filter-tags" ref={node}>
-      <input
-        type="text"
-        className="p-filter-tags__input"
+      <button
+        className="p-button has-icon"
         onClick={() => {
           setFilterPanelVisibility(true);
           sendAnalytics({
             category: "User",
             action: "Opened filter panel"
           });
         }}
-        placeholder={
-          activeFilters.length
+      >
+        <span className="p-filter-tags__active-count">
+          {activeFilters.length
             ? `Active filters: ${activeFilters.length}`
-            : "Filter models"
-        }
-      />
-
+            : "Filter models"}
+        </span>
+        <i className="p-icon--contextual-menu"></i>
```

Comment:
Can we not just use the vanilla component for this? https://docs.vanillaframework.io/patterns/contextual-menu#indicator